src/main/java/org/cyclonedx/maven/ModelConverter.java

```diff
@@ -46,9 +47,10 @@ public interface ModelConverter {
      * @param artifact the artifact to convert
      * @param schemaVersion the target CycloneDX schema version
      * @param includeLicenseText should license text be included in bom?
+     * @param externalReferences the external references
      * @return a CycloneDX component
      */
-    Component convert(Artifact artifact, CycloneDxSchema.Version schemaVersion, boolean includeLicenseText);
+    Component convert(Artifact artifact, CycloneDxSchema.Version schemaVersion, boolean includeLicenseText, ExternalReference[] externalReferences);
```

Comment:
This is a public API, can you deprecate the old method and provide a default implementation for it?

Reply:
This API is only used by the plugin itself. If you would look at the file history, the very last change before mine (i.e., 30d6af90e49962177564fc68289d4481c9fd3d7f by @hboutemy) also contains a breaking change.

Reply:
I agree that almost certainly nobody else uses it, but you know how conservative I am regarding BC.